e2e/helm_charts_test.go

```diff
@@ -31,17 +30,22 @@ func init() {
 // HelmChartTestSuite represents a test suite for a helm chart
 //nolint:unused
 type HelmChartTestSuite struct {
-	Name    string // the name of the chart
-	Version string // the helm chart version for the test
+	ClusterName string // the name of the cluster
+	Name        string // the name of the chart
+	Version     string // the helm chart version for the test
 }
 
 func (ts *HelmChartTestSuite) aClusterIsRunning() error {
-	args := []string{"status"}
+	args := []string{"get", "clusters"}
 
-	output, err := shell.Execute(".", "minikube", args...)
+	output, err := shell.Execute(".", "kind", args...)
```

Comment:
Thanks @kuisathaverat for the suggestion about kind!
![image](https://media.giphy.com/media/42YlR8u9gV5Cw/giphy.gif)